### ISSUE TITLE
User can create "sets" of dice.

### ISSUE THREAD

**Issue by Zaskoda** [Partavate-Studios/nft-tabletop-dice#68]
# Issue Description

Users will want to be able to switch between sets of dice depending on what game they are playing.

# Specific Requirements

- Sets are tracked within the client using cookies.
- Client provides quick interface for changing sets of dice.
- Sets of dice should be nameable. 
- The max number in a set should be higher than the current 3 (research and update this ticket when we decide on a max)

optional: dice sets could be 'savable' to the network for a fee

# Relevant Resources

**Comment by excalq** [comment]
Where do you see the business logic for this existing? In the contract, in the client, or in the minting scripts? It sounds like this is mainly in the client, given the cookies?

**Comment by Zaskoda** [comment]
Yes, in the client with cookies. Optionally, we could have a save button that stores the data to the network but that would require a fee from the user to do so. We could even enable this later with a different contract if need be.
